pkg/registry/core/service/storage/rest_test.go

```diff
@@ -2191,25 +1495,14 @@ func TestServiceRegistryExternalTrafficHealthCheckNodePortUserAllocation(t *test
 // Validate that the service creation fails when the requested port number is -1.
 func TestServiceRegistryExternalTrafficHealthCheckNodePortNegative(t *testing.T) {
```

Comment:
Not needed. Tested here: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go#L4528